src/libtriton/format/pe/pe.cpp

```diff
@@ -0,0 +1,204 @@
+//! \file
+/*
+**  Copyright (C) - Triton
+**
+**  This program is under the terms of the BSD License.
+*/
+
+#include <cstdio>
+#include <new>
+
+#include <pe.hpp>
+#include <exceptions.hpp>
+
+
+
+namespace triton {
+  namespace format {
+    namespace pe {
+
+      PE::PE(const std::string& path) {
```

Comment:
To get the same convention style than all others classes, the class `PE` should be `Pe`.